product/crud.py

```diff
@@ -0,0 +1,32 @@
+from .models import Plans
+from .schemas import PlansBase
+from sqlalchemy.orm.session import Session
+
+
+def create_plans(database_session: Session, request: PlansBase):
+
+    new_plan = Plans(
+        name = request.name,
+        description = request.description,
+        duration = request.duration,
+        price  = request.price
+    )
+    database_session.add(new_plan)
+    database_session.commit()
```

Comment:
I think we should all us db for a uniform work

Reply:
okay